dependencies/common/install-sccache

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+
+#
+# install-sccache
+#
+# Copyright (C) 2023 by RStudio, PBC
```

Comment:
```suggestion
# Copyright (C) 2023 by Posit Software, PBC
```

Sorry, I should've caught this the first time! This also needs to be Posit Software. I just `grep`-ed the repo and it looks like the majority of copyrights were updated but a few in `dependencies` were missed.

Reply:
Went ahead and updated the copyright notices in `dependencies` to use Posit!

Reply:
Nice, thank you!